<?xml version="1.0" ?>
<launch>
  <arg name="world_frame" default="kl2000_base_link"/> 
  <arg name="course_filename" default="/examples/ExampleAndreFlat.txt"/>
  <arg name="distance_waypoints" default="0.00574"/> <!--0.00574--> <!-- meters -->
  <arg name="smooth_start" default="true"/> <!-- m/s -->
  <arg name="add_x_distance" default="0.0"/> <!-- meters. Positive goes to the front, futher from the base-->
  <arg name="add_y_distance" default="0.0"/> <!-- meters. Positive goes to the left-->
  <arg name="add_z_distance" default="0.7"/> <!-- meters. Positive goes higher -->
  <arg name="tow_width" default="0.0"/>   <!-- inches -->
  <arg name="tow_thickness" default="0.001"/>   <!-- inches -->

  <node name="pose_builder_python" pkg="vsl_utilities" type="pose_builder.py" output="screen">
    <param name="world_frame" value="$(arg world_frame)"/>
    <param name="course_filename" value="$(arg course_filename)"/>  
    <param name="distance_waypoints" value="$(arg distance_waypoints)"/> 
    <param name="add_x_distance" value="$(arg add_x_distance)"/> 
    <param name="add_y_distance" value="$(arg add_y_distance)"/> 
    <param name="add_z_distance" value="$(arg add_z_distance)"/> 
    <param name="tow_width" value="$(arg tow_width)"/>  
    <param name="tow_thickness" value="$(arg tow_thickness)"/> 
    <param name="smooth_start" value="$(arg smooth_start)"/>  
  </node>

  <node name="course_display" pkg="vsl_utilities" type="course_display" output="screen"/>

</launch>